pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -58,6 +58,10 @@ import (
 // ReconcileTime is the delay between reconciliations
 const ReconcileTime = 30 * time.Second
 
+// ticket to redundant those vars - CSI-3071
```

Comment:
to remove*

Reply:
done

Reply:
I don't see it

Reply:
now it should be applied, I didn't write the done as part of the review